<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CUartCfg</class>
 <widget class="QWidget" name="CUartCfg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>835</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数字量模块配置</string>
  </property>
  <property name="windowIcon">
   <iconset resource="CgeneralControlSoftware.qrc">
    <normaloff>:/新前缀/img/logo.png</normaloff>:/新前缀/img/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
font: 12pt &quot;SimSun&quot;;
}
QPushButton{
	background-color: rgb(255, 255, 255);
	border-radius: 10px;
	
	font: 11pt &quot;SimSun&quot;;
	border: 1px solid rgb(150,150,150);
	
}

QPushButton:hover {
	background-color: rgb(200, 200, 200);
}

 QPushButton:pressed
{
	background-color: rgb(255, 255, 255);
}
</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>41</y>
     <width>501</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3,1,2">
    <item>
     <widget class="QLabel" name="labelCfgChannel">
      <property name="text">
       <string>配置通道：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="combCfgChannel">
      <property name="minimumSize">
       <size>
        <width>230</width>
        <height>28</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>第1路帧格式</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>第2路帧格式</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>第3路帧格式</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>第4路帧格式</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>第5路帧格式</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>第6路帧格式</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>第7路帧格式</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>第8路帧格式</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnCfgApplyToAll">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>28</height>
       </size>
      </property>
      <property name="text">
       <string>应用到所有</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>780</y>
     <width>441</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,0,3,0,3,0,3">
    <item>
     <widget class="QPushButton" name="btnOpen">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>28</height>
       </size>
      </property>
      <property name="text">
       <string>打开</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnSaveAs">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>28</height>
       </size>
      </property>
      <property name="text">
       <string>另存为</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnSave">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>28</height>
       </size>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>28</height>
       </size>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widgetChannelData" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>551</width>
     <height>661</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>7</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="labelCfgChannel_2">
           <property name="text">
            <string>数字量接口类型：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelCfgChannel_3">
           <property name="text">
            <string>接口工作模式：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelCfgChannel_5">
           <property name="text">
            <string>接口波特率：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelCfgChannel_8">
           <property name="text">
            <string>接口校验类型：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelCfgChannel_9">
           <property name="text">
            <string>接口停止位长：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>7</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QComboBox" name="combInterfaceType">
           <property name="minimumSize">
            <size>
             <width>230</width>
             <height>28</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>差分发送/差分接收</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>差分发送/光耦接收</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combWorkMode">
           <property name="minimumSize">
            <size>
             <width>230</width>
             <height>28</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>脉冲响应发送模式</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>命令响应发送模式</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>周期发送模式</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>脉冲取数模式</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>命令取数模式</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>被动接收工作模式</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>不工作</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combBaudRate">
           <property name="minimumSize">
            <size>
             <width>230</width>
             <height>28</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>921.6 kbs</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>460.8 kbs</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>230.4 kbs</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>115.2 kbs</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combCheckType">
           <property name="minimumSize">
            <size>
             <width>230</width>
             <height>28</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>无校验</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>奇校验</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>偶校验</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combStopBitLen">
           <property name="minimumSize">
            <size>
             <width>230</width>
             <height>28</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>1 位</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1.5 位</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2 位</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioDoubleRedundancy">
           <property name="text">
            <string>双冗余热备份</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="widget1SendPause_SendCmd_Send_ReceiveCmd" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="labelCfgChannel_7">
         <property name="text">
          <string>发送数据长度：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinSendDataLen">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>576</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget2SendPause_SendCmd_Send" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="labelCfgChannel_6">
         <property name="text">
          <string>帧计数位置：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinFrameNumPos">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1024</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget3SendPause_SendCmd_Send" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="labelCfgChannel_11">
         <property name="text">
          <string>数字量信息类型：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushDigitalEditor">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string>编辑</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget4SendPause_Receive" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <property name="spacing">
        <number>7</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="labelCfgChannel_12">
         <property name="text">
          <string>取数脉冲形式：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboPauseType">
         <item>
          <property name="text">
           <string>正脉冲</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>负脉冲</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget5SendPause_ReceivePause" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="labelCfgChannel_13">
         <property name="text">
          <string>取数脉冲宽度：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinPauseWidth">
         <property name="suffix">
          <string>μs</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget6SendCmd" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,1">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="labelCfgChannel_14">
         <property name="text">
          <string>命令帧帧头信息：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineCMDHeaderFormat">
         <property name="text">
          <string>00000000</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget7Send" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="labelCfgChannel_15">
         <property name="text">
          <string>数字量输出周期：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinDigitOutPer">
         <property name="suffix">
          <string>ms</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget8ReceivePause" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="labelCfgChannel_16">
         <property name="text">
          <string>取数脉冲输出间隔：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinPauseOutPer">
         <property name="suffix">
          <string>ms</string>
         </property>
         <property name="prefix">
          <string/>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget9ReceiveCmd" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="labelCfgChannel_17">
         <property name="text">
          <string>命令帧时间间隔：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinCmdOutPer">
         <property name="suffix">
          <string>ms</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget10ReceiveCmd" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="labelCfgChannel_18">
         <property name="text">
          <string>命令帧发送次数：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinCmdOutCount">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100000</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget11ReceiveCmd" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="labelCfgChannel_19">
         <property name="text">
          <string>命令帧1信息：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushCmdEditor1">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string>编辑</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget12ReceiveCmd" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="labelCfgChannel_20">
         <property name="text">
          <string>命令帧2信息：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushCmdEditor2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string>编辑</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget13ReceivePause_ReceiveCmd_Receive" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_16">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="labelCfgChannel_21">
         <property name="text">
          <string>接收数据长度：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinReceiveDataLen">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1024</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget14ReceivePause" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_17">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="labelCfgChannel_10">
         <property name="text">
          <string>取数脉冲发送次数：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinPauseOutCount">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100000</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>9</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CgeneralControlSoftware.qrc"/>
 </resources>
 <connections/>
</ui>
